_includes/home/_latest-message.html

```diff
@@ -0,0 +1,11 @@
+<section class="container soft-half push-bottom">
+<p class="text-gray font-family-serif text-uppercase push-bottom">latest weekly message</p>
+<iframe width="782" height="438" src="https://www.youtube.com/embed/ltTiQosa5JY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<h1 class="text-gray font-family-condensed-extra text-uppercase flush-top">Does Jesus keep his promises? </h1>
```

Comment:
Semantically speaking, this should not be an `<h1>` since it competes with other higher-priority content on the page.